test/new-e2e/pkg/environments/aws/docker/host.go

```diff
@@ -129,8 +130,8 @@ func Provisioner(opts ...ProvisionerOption) e2e.TypedProvisioner[environments.Do
 		if err != nil {
 			return err
 		}
-
-		host, err := ec2.NewVM(awsEnv, params.name, params.vmOptions...)
+		vmOptions := append(params.vmOptions)
```

Comment:
What's the goal of this line? It does not copy `params.vmOptions` and we don't need to copy it anyway.

Reply:
This is a leftover, because I was using `WithInstanceProfile` to set the instance because it was not defaulted in the VM creation at the beginning

Reply:
Removed